README.md

```diff
@@ -30,3 +30,25 @@ that belong in the OEM partition. Any RDEPENDS from these packages should
 be copied to the RDEPENDS in `board-packages` to ensure they are built.
 
 `coreos-base/coreos-oem-*` are metapackages for OEM specific ACIs. 
+
+# Updating
+
+To update follow the following steps:
+
+* Remove or rename the whole folder of the package to prepare the import from
+  upstream Gentoo, not only resetting the ebuild file but also any additional
+  files like patches or downstream additions under `files`.
+* Run `~/trunk/src/scripts/update_ebuilds --portage_stable . CATEGORY/PACKAGE`
+  in the `coreos-overlay` folder to import a new version from upstream Gentoo.
+* Commit the changes with a message like `CATEGORY/PACKAGE: Sync from Gentoo`.
```

Comment:
since we just copy, could we also include a hash to the gentoo repo commit from which it was sync'ed?

Reply:
I'd commit the synced version first, and then have a commit with a comment about the hash.

Reply:
Having it in the body of the commit message would also work, or?
`git show update_ebuilds/master` will give the commit ID